manifests/init.pp

```diff
@@ -83,7 +83,7 @@
     Class['passenger::config']
   }
 
-  if type($include_build_tools) == 'string' {
+  if type3x($include_build_tools) == 'string' {
```

Comment:
I guess that ```type3x``` will be supported much longer [1] than needed. If this should be a blocker, I will happily change it to use ```is_string()``` instead. Just drop me a line.

[1] https://github.com/puppetlabs/puppetlabs-stdlib#limitations